docs/docs/concepts/architecture.md

```diff
@@ -84,6 +84,8 @@ Cons:
 
 Raccoon sends the acknowledgments after the events are acknowledged successfully from the Kafka brokers. This configuration is recommended when reliable end-to-end acknowledgements are required. Here the underlying publisher acknowledgement is leveraged.
 
+![](/assets/raccoon_async.png)
```

Comment:
Q: Are the ack changes exclusively for `websockets`, or do we also support `REST/gRPC`?

Reply:
@AkbaraliShaikh ack changes are for all supported connection types